plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -160,16 +160,17 @@ private static JSONObject generateRootConfiguratorObject() {
         return rootConfiguratorObject;
     }
 
-    private static void generateMultipleAttributeSchema(JSONObject attributeSchema, Attribute attribute, BaseConfigurator baseConfigurator) {
+    private static void generateMultipleAttributeSchema(JSONObject attributeSchema, Attribute attribute, Object configObject) {
         if (attribute.type.getName().equals("java.lang.String")) {
-            ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
             attributeSchema.put(attribute.getName(),
                 new JSONObject()
                     .put("type", "string"));
         } else {
             ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
             try {
-                System.out.println("DocString: " + configurationAsCode.getHtmlHelp(baseConfigurator.getTarget(), attribute.getName()));
+
+                System.out.println("DocString: " + configurationAsCode.getHtmlHelp(Jenkins.getInstance().getDescriptor().getClass()
```

Comment:
@timja This too returns an empty string.
PS: The documentation uses the configurator object if i am not wrong...

Reply:
@timja The descriptor is kind of not returning anything.

Reply:
Have you tried debugging it or whatever?

Reply:
Yeah it seems that the resource is null, hence it returns an empty string https://github.com/jenkinsci/configuration-as-code-plugin/blob/fd433295bae57aceb9c69b853e9086716299cad0/plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java#L778

Reply:
`Jenkins.getInstance().getDescriptor().getClass()` 

`Jenkins` is not the best example for getting a descriptor. I doubt `Jenkins` class has a help file 😆

Reply:
I tried quite some stuff, any ideas on how to get it working?

Reply:
@casz
`configurationAsCode.getHtmlHelp(attribute.type.getClass(), attribute.getName()) ?`
This does not work still returning null :(